src/@carbon-info/views/Ecosystem/components/AppsCarousel.tsx

```diff
@@ -34,13 +34,15 @@ const AppsCarousel: React.FC<Props> = (props: Props) => {
         setView(currSlideMod);
       }}
       minimumTouchDrag={150}
+      transitionDuration={1000}
+      customTransition={"transform 1500ms cubic-bezier(.1,1.03,1,1) 0ms"}
     >
       {items.map((item, index) => {
         const { name, icon, description, tag, ctaLink } = item;
         const totalCards = items.length;
         const preview = widthSm ? 1 : 2;
         return view === index ? (
-          <Grow in timeout={(index + 1) * 200 > 1000 ? 1000 : (index + 1) * 200} key={`${name}-featured-dApps`} style={{ backgroundImage: `url("${item?.backgroundImage}")` }}>
+          <Grow in timeout={(index + 1) * 200 > 1000 ? 2000 : (index + 1) * 2000} key={`${name}-featured-dApps`} style={{ backgroundImage: `url("${item?.backgroundImage}")` }}>
```

Comment:
Think this implementation is not correct: this is causing the cards to appear very sluggishly.